COMFIE/Resources/DesignSystem/Components/CFNavigationBar/CFNavigationBar.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  CFNavigationBar.swift
+//  COMFIE
+//
+//  Created by Anjin on 3/27/25.
+//
+
+import SwiftUI
+
+struct CFNavigationBarButton: Identifiable {
+    let id: UUID = UUID()
+    let action: () -> Void
+    let label: AnyView
+    
+    init<Content: View>(
+        action: @escaping () -> Void,
+        @ViewBuilder label: () -> Content
+    ) {
+        self.action = action
+        self.label = AnyView(label())
+    }
+}
+
+struct CFNavigationBar: View {
+    let title: String
+    let isBackButtonHidden: Bool
+    let backButtonAction: () -> Void
+    let leadingButtons: [CFNavigationBarButton]
+    let trailingButtons: [CFNavigationBarButton]
+    
+    @Environment(\.dismiss) private var dismiss
+    
+    var body: some View {
+        ZStack {
+            // 네비게이션바 타이틀
+            Text(title)
+                .comfieFont(.title)
+                .foregroundStyle(Color.textBlack)
+            
+            HStack(spacing: 0) {
+                // 왼쪽 버튼들
+                HStack(spacing: 8) {
+                    if isBackButtonHidden == false {
+                        backButton()  // 뒤로가기 버튼
+                    }
```

Comment:
```suggestion
                    if !isBackButtonHidden {
                        backButton()  // 뒤로가기 버튼
                    }
```
요것도 소소한 의견입니다~ 
`== false`보다는` !`를 쓰는 게 더 간결해서 제안드려요 :)

Reply:
흐흐 요거 정말 취향 차이일 것 같은데요! 저는 개인적으로 '읽히는' 코드가 가독성이 좋다고 생각해서 ㅎ흐흐흫,, 일부러 !로 Bool 값을 바꾸어 사용하지 않는 편입니다만.. 흠 ㅎㅎ....